.github/workflows/release.yml

```diff
@@ -113,6 +114,27 @@ jobs:
           artifactid: 'ecs-logging-core'
           version: ${{ inputs.version }}
 
+  create-github-release:
+    name: "Create GitHub Release"
+    needs:
+      - await-maven-central-artifact
+    runs-on: ubuntu-latest
+    if: inputs.dry_run == false
```

Comment:
Ideally, https://github.com/cli/cli/issues/8887 could help with `dry-run` so this step could run regardless... though it might take a bit of time until agreed or implemented.